pages/index.tsx

```diff
@@ -1,19 +1,17 @@
 import Image from 'next/image'
 import Link from 'next/link'
 import MetaData from '../components/MetaData'
-import useTranslation from 'next-translate/useTranslation'
 
 const Index = () => {
-  const { t } = useTranslation('common')
```

Comment:
I thinking it might be a good idea to keep this so ALL the text stays in one place; add a common.json to the "default" language to pull the text?

Reply:
Next-i18next actually ignores the default locale folder entirely so this wouldn't work. You'd see `meta.title` as the text for the title for example. This wasn't always the case but it was a feature they implemented to work with how Next handles prefixing default locales.

See more [here](https://github.com/i18next/next-i18next/commit/f9a653966cd52fbb50bcbbd545369dbad50b489b)

That said, I do need to delete the default folder under locales since it literally does nothing.

Reply:
ahh sucky, how would you feel about a direct import of a .json file (obj) into here just for those values?

Reply:
That would work, sure.